Ginger/Ginger/AutomatePageLib/AddActionMenu/ApplicationModels/POMNavPage.xaml.cs

```diff
@@ -99,11 +99,32 @@ public POMNavPage(Context context)
 
             mContext.PropertyChanged += MContext_PropertyChanged;
             mPOMPage.OnSelect += MainTreeView_ItemSelected;
+
+            //mPOMPage.xTreeView.xTreeViewTree.ItemDoubleClick += POMItemsTreeView_ItemDoubleClick;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4507606)